src/components/VideoChat/VideoChat.tsx

```diff
@@ -131,9 +132,14 @@ export class VideoChat extends React.Component<VideoChatProps> {
       ourStream.getAudioTracks()[0].enabled =
         !ourStream.getAudioTracks()[0]?.enabled;
     }
+    this.debouncedEmitUserMute();
     this.forceUpdate();
   };
 
+  debouncedEmitUserMute = debounce(() => {
```

Comment:
Why's the debounce necessary?

Reply:
I had added it to avoid unnecessary traffic, considering how fast the button can be clicked. But I guess it was a bit inconsistent because other similar functions are not debounced/throttled either.